api/src/opentrons/config/advanced_settings.py

```diff
@@ -232,8 +232,8 @@ class Setting(NamedTuple):
     ),
     SettingDefinition(
         _id="disableOverpressureDetection",
-        title="Disable overpressure detection on pipettes.",
-        description="This setting disables overpressure detection on pipettes, do not turn this feature off unless recommended.",
+        title="Disable pipette pressure sensing.",
+        description="This setting disables pressure sensing capability on pipettes. This will prevent errors when overpressuring the pipette. Do not set this setting unless you are intentionally causing >8kPa inside the pipette air channel.",
```

Comment:
Because we have puny little monkey brains that are bad at negation, "turn this switch **on** to **disable** the thing" can be confusing. Suggested text for flipping the polarity of the switch:
```suggestion
        title="Pipette pressure sensing",
        description="By default, Flex stops and displays an error when it detects high pressure inside a pipette. Only turn this setting off if you are intentionally causing pressures over 8 kPa inside the pipette air channel.",
```

This parallels the structure for explaining the Home Gantry on Restart setting that we have in the Flex manual (which I think also went through a similar polarity flip during its development):
> By default ,the gantry moves to its home position any time you turn on Flex. Only disable this behavior if you have a reason that the gantry must remain stationary after powering on.